addon/styles/addon.scss

```diff
@@ -39,19 +42,25 @@
   align-items: flex-end;
 }
 
+
 .frost-file-picker .frost-text,
 .frost-file-picker .file-select-overlay .text {
+  transition: border-bottom-color 1s ease;
   width: 100%;
 }
 
 .frost-file-picker .frost-text:read-only {
-  border-bottom: 1px solid $frost-color-grey-6;
+  border-bottom-color: $frost-color-grey-6;
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
```

Comment:
For transitions. You can't just change the color, it has a weird effect when you ease in border